easybuild/easyconfigs/s/SCOTCH/SCOTCH-7.0.3-gompi-2022b.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'CMakeMake'
```

Comment:
Maybe we should update the custom easyblock for `SCOTCH` instead, to switch to CMake for versions >= 7.0.3?

Reply:
Although it should be possible to update scotch.py easyblock, the switch to generic CMakeMake is, under my opinion, a better choice. It will not be necessary to maintain the scotch.py easyblock. 

Nevertheless, the easybuild is your project, so let me know if you prefer to use custom easyblock over CMakeMake and I will try to update it.

Reply:
The easyconfig was changed to use SCOTCH easyblock.